### PR TITLE
NI: Replace Canal10

### PR DESCRIPTION
**Pull request by fershady**

### CODE REVIEW

**Review by dct-infra — CHANGES_REQUESTED** [review]
Would you please remove the [Timeout] tag from the #EXTINF  and change the res to 480p?

**Review by dct-infra — DISMISSED** [review]

File: streams/ni.m3u [hunk 1]
```diff
@@ -3,8 +3,8 @@
 http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/canal6nicaraguaoficial
 #EXTINF:-1 tvg-id="Canal9.ni" status="timeout",Canal 9 (480p) [Timeout]
 https://cootv.cootel.com.ni/streams/d/SSLCH09-HD-CVS/playlist.m3u8
-#EXTINF:-1 tvg-id="Canal10.ni" status="timeout",Canal 10 (576p) [Timeout]
-https://cootv.cootel.com.ni/streams/d/SSLCH10-SD-ZUK/playlist.m3u8
+#EXTINF:-1 tvg-id="Canal10.ni" status="timeout",Canal 10 (480p) [Geo-blocked]
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.canal10.com.ni/envivo/
```
Comment (freearhey) [inline streams/ni.m3u:7]: The link no longer works

image

**Review by freearhey — CHANGES_REQUESTED** [review]

File: streams/ni.m3u (hunk 1 above)
Comment (fershady) [reply streams/ni.m3u:7]: The stream is only available for news. From 5:00 to 9:00, then from 13:00 to 14:00, and then from 19:00 to 20:00. [UTC-06:00]. The rest of the time is restricted.

Comment (freearhey) [reply streams/ni.m3u:7]: In this case it should be marked as `[Not 24/7]`

**Comment by freearhey** [comment]
@fershady Are you sure the link is geo-blocked as well?

**Comment by fershady** [comment]
Geo-blocked 
Screenshot 2022-02-25 at 1 08 05 PM

**Comment by fershady** [comment]
but now is live
Screenshot 2022-02-25 at 2 02 45 PM

**Review by freearhey — DISMISSED** [review]

**Review by its0x08 — DISMISSED** [review]

**Pull request closed without merging**